test/AccordionSpec.js

```diff
@@ -89,4 +89,12 @@ describe('<Accordion>', () => {
 
     expect(onClickSpy).to.be.calledOnce;
   });
+
+  it('Should define default "as" in prop destructuring instead of deafultProps', () => {
```

Comment:
i'm not sure these test cases really add much. i think it's fine as long as the old tests pass.

Reply:
You're right, I was mostly thinking about having them in order to prevent someone from inadvertently putting the "as" into the defaultProps at a later point. This whole thing can be considered a bit weird, so I thought this would be an ok way to show why they're set up like they are. What do you think?

Reply:
I think it'd be more effective to add a comment in the places where you set the default, honestly. Otherwise nothing stops someone from carelessly removing the test as well.